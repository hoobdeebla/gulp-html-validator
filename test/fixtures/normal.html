<!DOCTYPE html>
<html lang="en">
 <head>
   <meta charset=utf-8>
   <title>A simple HTML5 page layout</title>
   <!--[if IE]>
     <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js">
     </script>
   <![endif]-->
   <style>
     body { margin:100px; }
     article, aside, figure, footer, header, hgroup, menu, nav, section {
       display:block;
     }
   </style>
 </head>
 <body>
   <nav>
     <a href="simple-html5-document.html">home1</a>
     <a href="simple-html5-document.html">home2</a>
   </nav>
   <header>
     <h1>A simple HTML5 document</h1>
   </header>
   <article>
     <section>
       <h3>Section 1 heading</h3>
     </section>
   </article>
   <article>
     <header>
       <h1>Article 2 heading</h1>
     </header>
     <section>
       <h2>Section 1 heading</h2>
     </section>
   </article>
   <footer>
     <p>This is the footer</p>
   </footer>
 </body>
</html>
