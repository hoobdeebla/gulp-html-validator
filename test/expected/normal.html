{"messages":[{"type":"info","message":"The Content-Type was “text/html”. Using the HTML parser."},{"type":"info","message":"Using the schema for HTML with SVG 1.1, MathML 3.0, RDFa 1.1, and ITS 2.0 support."},{"type":"info","lastLine":25,"lastColumn":12,"firstColumn":4,"subType":"warning","message":"Article lacks heading. Consider using “h2”-“h6” elements to add identifying headings to all articles.","extract":"eader>\n   <article>\n     ","hiliteStart":10,"hiliteLength":9},{"type":"info","lastLine":32,"lastColumn":11,"firstColumn":8,"subType":"warning","message":"Consider using the “h1” element as a top-level heading only (all “h1” elements are treated as top-level headings by many screen readers and other tools).","extract":"r>\n       <h1>Articl","hiliteStart":10,"hiliteLength":4}]}
